_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -34478,7 +34478,7 @@
 /turf/open/floor/plasteel,
 /area/engine/break_room)
 "bdw" = (
-/obj/machinery/computer/card/minor/rd{
+/obj/machinery/modular_computer/console/preset/id{
```

Comment:
Are these a suitable replacement for the department ID consoles, which only update their own department?

Reply:
Logging on with the appropriate head ID looks like this:
![image](https://user-images.githubusercontent.com/13771064/106954839-f815cc00-66f9-11eb-9986-aecf24ea3e5c.png)